Description: Add Close Button
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change
 .
 qmlplane (1.1.0-1) unstable; urgency=medium
 .
   * Initial release
Author: JBBgameich <jbb.mail@gmx.de>

--- qmlplane-1.1.0.orig/qml/screens/MenuScreen.qml
+++ qmlplane-1.1.0/qml/screens/MenuScreen.qml
@@ -60,6 +60,13 @@ BlankScreen {
             onClicked:
                 stackView.push(Qt.resolvedUrl("AboutScreen.qml"))
         }
+        
+        Controls.Button {
+            text: "Close"
+            color: "#d76241"
+            onClicked:
+                appWindow.close()
+        }
     }
 
     Controls.Label {
